sbi2-2023-cytotable-poster/poster.rmd

```diff
@@ -0,0 +1,269 @@
+knitr::include_graphics("images/cytosnake_MP_processing_cropped.png")
+```
+<p class="figcaption">Figure 4. Diagram showing how CytoSnake orchestration applied in multiple pipelines.</p>
+
+<div class="container">
+  <div class="text-block"  style="padding-top:40px;">CytoSnake is an innovative tool for orchestrating high-dimensional cell morphology data processing pipelines, including those which leverage CytoTable and other applied usecases.</div>
+  <div class="text-block" style="text-align:center;"><img src="images/qr_cytosnake.png" style="width:70%"/></div>
+</div>
+
+
+## Applied research: NF1 Schwann cell project
+
+___Authors:___ Jenna Tomkinson, Cameron Mattson, Erik Serrano,<br> Gregory P. Way
+
+```{r, echo=FALSE, out.width='100%', fig.align='center'}
+knitr::include_graphics("images/nf1_and_cfret_workflow.png")
+```
+<p class="figcaption">Figure 5. Diagram showing NF1 pipeline implementation details including CytoTable and Pycytominer.</p>
+
+<div class="container">
+  <div class="text-block"  style="padding-top:40px;">Comprehensive dataset analyses for cell painting assays, enabling the further understanding of cellular morphology NF1 Schwann cells and rare disease treatment.</div>
+  <div class="text-block" style="text-align:center;"><img src="images/qr_nf1.png" style="width:70%"/></div>
+</div>
+
+<br>
+<br>
+<br>
+
+## Applied research: Pyroptosis signature project
+
+___Authors:___ Michael J. Lippincott, Jenna Tomkinson, Gregory P. Way
+
+```{r, echo=FALSE, out.width='100%', fig.align='center'}
+knitr::include_graphics("images/Cell_Painting_Figure_-_Interstellar_Poster_1.png")
+```
+
+<p class="figcaption">Figure 6. Diagram showing pyroptotic cell changes which are studied as part of the Pyroptosis signature project.</p>
+
+<div class="container">
+  <div class="text-block"  style="padding-top:40px;">Identifying and characterizing pyroptosis signatures in cellular systems, aiding in the study of inflammatory cell death pathways as part of the Interstellar collaboration.</div>
+  <div class="text-block" style="text-align:center;"><img src="images/qr_pyroptosis.png" style="width:70%"/></div>
+</div>
+
+
+## Applied research: CFReT project
+
+___Authors:___ Jenna Tomkinson, Erik Serrano, Gregory P. Way
+
+```{r, echo=FALSE, out.width='100%', fig.align='center'}
+knitr::include_graphics("images/nf1_and_cfret_workflow.png")
+```
+<p class="figcaption">Figure 7. Diagram showing CFReT pipeline implementation details including CytoTable and Pycytominer.</p>
+
+<div class="container">
+  <div class="text-block"  style="padding-top:40px;">Image-based analysis of cardiac fibroblast datasets to uncover proprietary drug impact on reversing fibrosis.</div>
+  <div class="text-block" style="text-align:center;"><img src="images/qr_cfret.png" style="width:70%"/></div>
+</div>
+
+# Using CytoTable
+
+```python
+import cytotable
+result_file = cytotable.convert(
+    source_path="path/to/feature-data",
+    dest_path="destination/path.parqut",
```

Comment:
Tiny typo fix:

```suggestion
    dest_path="destination/path.parquet",
```

Reply:
Thanks - great catch! I'll commit this in.